schema/staking.sql

```diff
@@ -43,3 +43,14 @@ CREATE TABLE validator_unbonding_delegations
     timestamp         TIMESTAMP WITHOUT TIME ZONE NOT NULL,
     PRIMARY KEY (consensus_address, delegator_address, height)
 )
+
+CREATE TABLE validator_commission
+(
+    validator_address     CHARACTER VARYING(52) NOT NULL REFERENCES validator (consensus_address),
+    timestamp             TIMESTAMP WITHOUT TIME ZONE NOT NULL,
+    commission            decimal	              NOT NULL,
+    min_self_delegation    BIGINT              NOT NULL,
+    Discription          CHARACTER VARYING(52) NOT NULL,
+    height                BIGINT                NOT NULL,
+    PRIMARY KEY (validator_address,height)
+);
```

Comment:
Since the message is passed, we don't know whether is the commission changed or min self delegation/description changed. If the user changed the self delegation/description but not the commission rate, there are duplicated entries. So I think that's better to store the entries that can be changed by the MsgEditValidator. Though changing the name of the table should be a good idea?

Reply:
> Since the message is passed, we don't know whether is the commission changed or min self delegation/description changed. If the user changed the self delegation/description but not the commission rate, there are duplicated entries. 

You're right. I didn't think about it. I only got two observations about these points: 

1. We might not be interested in storing the historical changes of the validator description.  
   Usually the only data that is relevant for explorers is the latest description available. For this reason, I would just store it inside another table that does not change that frequenly and has at most one row per validator (this can be the `validator_info` table adding a column to it). 

2. To solve the duplication issue, what we might do is actually querying the database before saving the information.  
    If we just query the latest row of the database abased on the associated height and check if the commission rate is different from the one present inside `MsgEditValidator`, then we can decide to skip the storing. This would avoid duplicate entries when something other than the current commission rate value changes.